apps/crn-server/test/data-providers/contentful/analytics.data-provider.test.ts

```diff
@@ -176,14 +176,15 @@ describe('Analytics Data Provider', () => {
           expect(result.items[0]!.interestGroupLeadershipRoleCount).toBe(0);
         });
 
-        test('Should return interestGroupLeadershipRoleCount of 2 when two users are leaders of two different interest groups', async () => {
+        test('Should return interestGroupLeadershipRoleCount of 2 when two users are leaders of two different active interest groups', async () => {
```

Comment:
If the team is inactive (when we add `inactiveSince` in the fixture) like the screenshot below

<img width="706" alt="Screenshot 2024-03-29 at 12 54 17" src="https://github.com/yldio/asap-hub/assets/16595804/e984d27e-ed21-47ce-8ca3-08d358c58c44">

This test still passes although it should be 0